types/evidence.go

```diff
@@ -57,14 +57,13 @@ func (err *ErrEvidenceOverflow) Error() string {
 
 // Evidence represents any provable malicious activity by a validator
 type Evidence interface {
-	Height() int64                                     // height of the equivocation
-	ValidatorHeight() int64                            // height of validators
-	Time() time.Time                                   // time of the equivocation
-	Address() []byte                                   // address of the equivocating validator
-	Bytes() []byte                                     // bytes which comprise the evidence
-	Hash() []byte                                      // hash of the evidence
-	Verify(chainID string, pubKey crypto.PubKey) error // verify the evidence
-	Equal(Evidence) bool                               // check equality of evidence
+	Height() int64          // height of the equivocation
+	ValidatorHeight() int64 // height of validators
+	Time() time.Time        // time of the equivocation
+	Address() []byte        // address of the equivocating validator
+	Bytes() []byte          // bytes which comprise the evidence
+	Hash() []byte           // hash of the evidence
+	Equal(Evidence) bool    // check equality of evidence
```

Comment:
Removed Verify function as not all evidence can be verified which only the chain ID and validator public key